plugins/ethereum-crypto/scripts/freeze_dependencies.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
```

Comment:
this script is duplicated with the one in the root `scripts/freeze_dependencies`